includes/settings/class-wcpdf-settings-debug.php

```diff
@@ -493,16 +493,15 @@ private function reset_settings( $data ) {
 	public function ajax_process_danger_zone_tools() {
 		check_ajax_referer( 'wpo_wcpdf_debug_nonce', 'nonce' );
 		
-		$request = stripslashes_deep( $_REQUEST );
+		$request = stripslashes_deep( $_POST );
 		
 		if ( ! isset( $request['document_type'] ) || ! isset( $request['date_from'] ) || ! isset( $request['date_to'] ) ) {
 			$message = __( 'One or more request parameters missing.', 'woocommerce-pdf-invoices-packing-slips' );
-			wp_send_json_error( compact( $message ) );
+			wp_send_json_error( compact( 'message' ) );
 		}
 	
-		$from_date          = date_i18n( 'Y-m-d', strtotime( $request['date_from'] ) );
-		$to_date            = date_i18n( 'Y-m-d', strtotime( $request['date_to'] ) );
-		$date_type          = apply_filters( 'wpo_wcpdf_process_danger_zone_tools_date_type', 'date_created', $request );
+		$from_date          = strtotime( $request['date_from'] );
+		$to_date            = strtotime( $request['date_to'] );
```

Comment:
HPOS is disabled. I have 3 orders from the 19th to the 22nd.
Since the time is not given to the `$to_date`, it considers the time as `00:00:00`, and so a document that I have in "February 22, 2024, 1:26:34 PM" will not be considered to be renumbered. Maybe it's good to add `. ' 23:59:59'` at the end of this?